pkg/api/v1beta2/dynakube/convert_from.go

```diff
@@ -29,7 +26,11 @@ func (dst *DynaKube) ConvertFrom(srcRaw conversion.Hub) error {
 	return nil
 }
 
-func (dst *DynaKube) fromBase(src *v1beta2.DynaKube) {
+func (dst *DynaKube) fromBase(src *v1beta3.DynaKube) {
```

Comment:
Please apply fixes from the JIRA ticket

Reply:
I did, you were too quick :)  fixed in 6fb60db3